File: python/ejer51.py
notas = [4.0, 3.5, 2.5, 5.0, 4.2]

promedio = sum(notas)/len(notas)

print("Promedio =", promedio)

if promedio >= 3:
    print("Aprobado")
else:
    print("Reprobado")

File: js/ejer51.js
let notas = [4.0, 3.5, 2.5, 5.0, 4.2];

let suma = 0;

for(let i=0;i<notas.length;i++){
    suma += notas[i];
}

let promedio = suma / notas.length;

console.log("Promedio =", promedio);

if(promedio >= 3){
    console.log("Aprobado");
}else{
    console.log("Reprobado");
}